### PR TITLE
[FE-25] Create script to GET requester pays status of a workspace bucket

### PR DESCRIPTION
**Pull request by samanthahv**
https://broadworkbench.atlassian.net/browse/FE-25

Example output for disabled RP bucket:
image

Example output for enabled RP bucket:
image

### CODE REVIEW

**Comment by mmorgantaylor** [comment]
looks good! can you demonstrate some testing by running the script on both a RP and non-RP bucket and showing the script's output in the PR description?

**Comment by samanthahv** [comment]
@mmorgantaylor i added a screenshot of the output for a non-RP bucket to the Jira ticket, but i can add it to this PR description along with a screenshot for a RP-enabled bucket

**Comment by samanthahv** [comment]
@mmorgantaylor done!

**Review by mmorgantaylor — CHANGES_REQUESTED** [review]
a couple questions/requests!

**Comment by samanthahv** [comment]
addressed your comments in this commit [46cf4e3](https://github.com/broadinstitute/horsefish/pull/135/commits/46cf4e3a27f2840aa64d0e29840f56e87ebfa497)

File: scripts/get_requester_pays_status.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+if (( $# < 2 )); then
+  echo "Usage: $0 TERRA_PROJECT_ID PATH_TO_TERRA_BUCKET_PATH"
+  echo "The Terra bucket path can be formatted as gs://fc-XXXXX or fc-XXXXX"
+  echo "i.e you can run ./set_requester_pays.sh on project_id fc-12345"
```
Comment (mmorgantaylor) [inline scripts/get_requester_pays_status.sh:6]: change to `./get_requester_pays.sh`

Comment (mmorgantaylor) [reply scripts/get_requester_pays_status.sh:6]: also remove the word "on" in the command and add backticks - so I think this line should be 
```echo "i.e you can run `./get_requester_pays.sh project_id fc-12345`"```

**Pull request merged**